src/components/BigPicture/hooks/useControllerNavigation.ts

```
@@ -0,0 +1,184 @@
+  return Array.from(
+    document.querySelectorAll<HTMLElement>('.bp-focusable'),
+  ).filter((el) => !el.closest('[data-bp-hidden]') && el.offsetParent !== null);
+};
+
+const getRow = (el: HTMLElement) => el.dataset.bpRow ?? '';
+
+// Focuses an element without the browser's instant scroll-jump, then
+// smoothly scrolls the element into view via the scroll container.
+const focusAndScroll = (el: HTMLElement) => {
+  el.focus({ preventScroll: true });
+  el.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'nearest' });
```

ESLint

[error] Replace `·behavior:·'smooth',·block:·'nearest',·inline:·'nearest'` with `⏎····behavior:·'smooth',⏎····block:·'nearest',⏎····inline:·'nearest',⏎·`

(prettier/prettier)